src/Workspaces/Core/Portable/Options/EditorConfigStorageLocation.cs

```diff
@@ -77,19 +77,25 @@ public EditorConfigStorageLocation(string keyName, Func<string, object> parseVal
         public EditorConfigStorageLocation()
         {
             // If the user didn't pass a keyName assume we need to parse the entire dictionary
-            _tryParseDictionary = (dictionary, type) =>
+            _tryParseDictionary = (dictionary, type, underlyingOption) =>
```

Comment:
The user can pass in their own function to be used.  This is  only used in the case where the user does not supply one

---
In reply to: [96749579](https://github.com/dotnet/roslyn/pull/16510#discussion_r96749579) [](ancestors = 96749579)